engine/lib/river.php

```diff
@@ -1,4 +1,6 @@
 <?php
+use Elgg\Exceptions\InvalidArgumentException;
+
```

Comment:
please insert below file description

Reply:
also make this an alias like ``ElggInvalidArgumentException`` to prevent notices from Scrutinizer